style.css

```diff
@@ -0,0 +1,101 @@
+@media only screen and (max-width: 600px) {
+  body {
+    background-color: black;
+    max-height: 1000px;
+  }
+
+  header {
+    background-color: black;
+  }
+
+  .home {
+    background-image: url("images/Group.png");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: bottom right;
+  }
```

Comment:
While you were giving a `flex` display for some elements, I recommend you to add `flex` and `column` to the home class element, so it's responsive on your webpage, since giving flex to only one element it's does anything.
- [ ] Kindly add a `flex` display to your home class element